src/group_theory/subgroup/basic.lean

```diff
@@ -1030,6 +1030,26 @@ instance bot_normal : normal (⊥ : subgroup G) := ⟨by simp⟩
 @[priority 100, to_additive]
 instance top_normal : normal (⊤ : subgroup G) := ⟨λ _ _, mem_top⟩
 
+/-- Group conjugation on a normal subgroup. Analogous to `mul_aut.conj`. -/
+def mul_aut.conj_normal [hH : H.normal] : G →* mul_aut H :=
+{ to_fun := λ g,
+  { to_fun := λ h, ⟨g * h * g⁻¹, hH.conj_mem h h.2 g⟩,
+    inv_fun := λ h, ⟨g⁻¹ * h * g, by rw [hH.mem_comm_iff, mul_inv_cancel_left]; exact h.2⟩,
+    left_inv := λ _, by simp [mul_assoc],
+    right_inv := λ _, by simp [mul_assoc],
+    map_mul' := λ x y, by ext; simp [mul_assoc] },
+  map_mul' := λ _ _, by ext; simp [mul_assoc],
+  map_one' := by ext; simp [mul_assoc] }
+
+@[simp] lemma mul_aut.conj_normal_apply [hH : H.normal] (g : G) (h : H) :
+  ↑(mul_aut.conj_normal g h) = g * h * g⁻¹ := rfl
+
+@[simp] lemma mul_aut.conj_normal_symm_apply [hH : H.normal] (g : G) (h : H) :
+  ↑((mul_aut.conj_normal g).symm h) = g⁻¹ * h * g := rfl
+
+@[simp] lemma mul_aut.conj_normal_inv_apply [hH : H.normal] (g : G) (h : H) :
+  ↑((mul_aut.conj_normal g)⁻¹ h) = g⁻¹ * h * g := rfl
```

Comment:
Might as well `to_additive` everything here:
```suggestion
@[to_additive "Group conjugation on a normal subgroup. Analogous to `add_aut.conj`"]
def mul_aut.conj_normal [hH : H.normal] : G →* mul_aut H :=
{ to_fun := λ g,
  { to_fun := λ h, ⟨g * h * g⁻¹, hH.conj_mem h h.2 g⟩,
    inv_fun := λ h, ⟨g⁻¹ * h * g, by rw [hH.mem_comm_iff, mul_inv_cancel_left]; exact h.2⟩,
    left_inv := λ _, by simp [mul_assoc],
    right_inv := λ _, by simp [mul_assoc],
    map_mul' := λ x y, by ext; simp [mul_assoc] },
  map_mul' := λ _ _, by ext; simp [mul_assoc],
  map_one' := by ext; simp [mul_assoc] }

@[simp, to_additive] lemma mul_aut.conj_normal_apply [hH : H.normal] (g : G) (h : H) :
  ↑(mul_aut.conj_normal g h) = g * h * g⁻¹ := rfl

@[simp, to_additive] lemma mul_aut.conj_normal_symm_apply [hH : H.normal] (g : G) (h : H) :
  ↑((mul_aut.conj_normal g).symm h) = g⁻¹ * h * g := rfl

@[simp, to_additive] lemma mul_aut.conj_normal_inv_apply [hH : H.normal] (g : G) (h : H) :
  ↑((mul_aut.conj_normal g)⁻¹ h) = g⁻¹ * h * g := rfl
```

Reply:
@Vierkantor This doesn't seem to work. In the additive case, you can't write `G →+ add_aut H`, because `add_aut H` is still multiplicative, I think.

Reply:
We solve that as `add_aut.conj : G →+ additive (add_aut G)`, but `to_additive` isn't smart enough for that.

Reply:
Should I still try to make everything `to_additive`? It looks like it's going to be pretty painful (seems like each simp lemma needs the additive version stated separately, and the `simp [mul_assoc]` proofs in the multiplicative case no longer work in the additive case because of the `additive (...)`, I think).

Reply:
I'm not convinced the additive case is useful anyway; we don't have an analog of `mul_distrib_mul_action` or `conj_act` for it.